labshare-client/src/views/Index.vue

```diff
@@ -6,7 +6,7 @@
           <b-col cols lg="6" md="12" sm="12">
             <h2>Wir bündeln Kräfte für mehr SARS-CoV-2 Tests</h2>
             <p>Gemeinsam entfesseln wir die geballte Kraft der Wissenschaft!</p>
-            <b-button href="#">Jetzt mitmachen</b-button>
+            <b-button class="btn-primary" href="#">Jetzt mitmachen</b-button>
```

Comment:
All the text here should be piped through the i18n infrastructure, but that is likely beyond the scope of this individual PR

Reply:
Out of scope for this, Lisa is currently doing the localization stuff. But we should use the bootstrap-vue attributes. So `variant="primary"` instead of `class="..."`.